js/scripts.js

```diff
@@ -0,0 +1,38 @@
+//scripts
+$(function() {
+var tweetLink = "https://twitter.com/intent/tweet?text=";
+var quoteUrl = "https://quotesondesign.com/wp-json/posts?filter[orderby]=rand&filter[posts_per_page]=1";
+
+var prefix = "https://cors-anywhere.herokuapp.com/";
+function getQuote() {
+    $.getJSON(prefix + quoteUrl, createTweet);
+    $.ajaxSetup({ cache: false });
+}
+
+function createTweet(input) {
+    var data = input[0];
+
+    var quoteText = $(data.content).text().trim();
+    var quoteAuthor = data.title;
+
+    if (!quoteAuthor.length) {
+        quoteAuthor = "Unknown author";
+    }
+
+    var tweetText = "Quote of the day - " + quoteText + " Author: " + quoteAuthor;
+    if (tweetText.length > 140) {
+    getQuote();
+	} else {
+    var tweet = tweetLink + encodeURIComponent(tweetText);
+    $('.quote').text(quoteText);
```

Comment:
jeżeli odwołujesz się do HTML w JS najlepiej jest to zrobić po ID a nie po klasie.
https://stackoverflow.com/questions/6460644/in-jquery-is-selecting-by-class-or-id-faster-than-selecting-by-some-other-attri